content/zh/docs/hertz/tutorials/basic-feature/route.md

```diff
@@ -64,7 +64,7 @@ func main(){
 	h.Any("/ping_any", func(ctx context.Context, c *app.RequestContext) {
 		c.String(consts.StatusOK, "any")
 	})
-	h.Handle("Load","/load", func(ctx context.Context, c *app.RequestContext) {
+	h.Handle("LOAD","/load", func(ctx context.Context, c *app.RequestContext) {
```

Comment:
same to https://github.com/cloudwego/cloudwego.github.io/commit/bbd644313b25ab553beef8e952f399e64358beb7 ??